app/src/main/res/values/strings.xml

```diff
@@ -398,6 +409,7 @@
     <string name="scope_published_title">Published</string>
     <string name="scope_published_description">Synchronize to trusted servers and the global and public address book</string>
     <string name="scope_toggle">Scope toggle</string>
+    <string name="scope_toggle_description">Scope toggle for %1$s (%2$s)</string>
```

Comment:
In the server we use a less technical wording here: "Change privacy level of […]"
Ref https://github.com/nextcloud/server/pull/26548/files#diff-c1bbc91dbbd993cc8d0ee7bcef864eb9290be2fa29dd4810d92f1f3a19449183

Reply:
DONE, thanks for the hint&link 👍